app/celery/process_ses_receipts_tasks.py

```diff
@@ -209,24 +210,9 @@ def process_ses_results(  # noqa: C901 (too complex 14 > 10)
             check_and_queue_va_profile_email_status_callback(notification)
 
             return
-
-        # Prevent regressing bounce status
-        if (
-            notification.status_reason
-            and 'bounce' in notification.status_reason
-            and notification.status
-            in {
-                NOTIFICATION_TEMPORARY_FAILURE,
-                NOTIFICATION_PERMANENT_FAILURE,
-            }
-        ):
-            # async from AWS means we may get a delivered status after a bounce, in rare cases
-            current_app.logger.warning(
-                'Notification: %s was marked as a bounce, cannot be updated to: %s',
-                notification.id,
-                notification_status,
-            )
-            return
```

Comment:
Why was this removed?

Reply:
It's redundant to this previous "if" block.  The only different is a slightly different logged message.  Both blocks end with "return".

Reply:
We discussed this.  I think I get it.  `notification.status` refer to the existing status; not the incoming, updated status.

Reply:
restored